x-pack/plugins/triggers_actions_ui/public/application/components/add_message_variables.scss

```diff
@@ -0,0 +1,4 @@
+.messageVariablesPanel {
+  max-height: 400px;
+  overflow-y: scroll;
```

Comment:
Check the docs on how to handle overflows. https://elastic.github.io/eui/#/guidelines/sass

You can probably do this by using one of the utility classes, or using the mixin. That way you only need the max height.

Reply:
Essentially just...

```suggestion
  @include euiYScrollWithShadows;
```